src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/GuardArrow.java

```diff
@@ -61,6 +63,14 @@ protected void arrowHit(final EntityLivingBase targetEntity)
         Log.getLogger().info("Arrow hit " + targetEntity + " with " + targetEntity.getHealth());
         if (targetEntity.getHealth() <= 0.0F)
         {
+            if (targetEntity instanceof EntityPlayer)
+            {
+                EntityPlayer player = (EntityPlayer) targetEntity;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)